### PR TITLE
Add option in my_user_config.h to turn on SDM630 import active energy

### PR DESCRIPTION
**Pull request by jkostorz**
## Description:

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [x] The pull request is done against the latest dev branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR.
  - [x] The code change is tested and works on Tasmota core ESP8266 V.2.7.4.1
  - [ ] The code change is tested and works on core ESP32 V.1.12.2
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

File: tasmota/my_user_config.h [hunk 1]
```diff
@@ -637,6 +637,7 @@
   #define SDM120_SPEED         2400              // SDM120-Modbus RS485 serial speed (default: 2400 baud)
 //#define USE_SDM630                               // Add support for Eastron SDM630-Modbus energy monitor (+0k6 code)
   #define SDM630_SPEED         9600              // SDM630-Modbus RS485 serial speed (default: 9600 baud)
+//#define SDM630_IMPORT                          // Show import active energy in MQTT and Web (+0k3 code)
```
Comment (eku) [inline tasmota/my_user_config.h:640]: Does this setting depend on ```USE_SDM630``` somehow?

Comment (jkostorz) [reply tasmota/my_user_config.h:640]: By default, Tasmota reads multiple ModBus registers from SDM630 but without 0x015A, 0X015C, 0x015E (phase 1, 2 and 3 import active energy). I have been using the extended version that reads these registers for four months now and it works very well. For users who need to read the consumed energy separately for each phase, there is this switch - #SDM630_IMPORT. This works ONLY if the SDM630 is compiled in the firmware as before (tasmota-sensors etc).

Comment (eku) [reply tasmota/my_user_config.h:640]: Well, I mean, does the code compile if ```SDM630_IMPORT``` is defined but ```USE_SDM630``` is not?

Comment (jkostorz) [reply tasmota/my_user_config.h:640]: SDM630_IMPORT work only if USE_SDM630 is defined.

**Pull request merged**